### DISCUSSION TITLE
Conbee II firmware

### DISCUSSION THREAD

**Discussion by Vandborg** [Koenkk/zigbee2mqtt#8195] [Devices]
Hi maybe I'm just bonkers here? But is it correct if the coordinator link for the firmware to flash to the conbee II stick is wrong? If yes, could anyone point me to the correct firmware link?

[https://www.zigbee2mqtt.io/information/supported_adapters.html#overview](https://www.zigbee2mqtt.io/information/supported_adapters.html#overview)

**Comment by c0mplex1** [comment]
This link is okay.
After click on the link, scroll down till you see the ConBee II stick, then click on "Flashing" button.
Now you can follow the "Update deCONZ manually" procedure.

**Reply by Vandborg** [reply]
You misunderstand me "https://www.zigbee2mqtt.io/information/supported_adapters.html#overview" works just fine. It's the coordinator link that is wrong I believe. It leads to the same place as flashing. So I know how to flash the conbee stick. I just don't have the firmware to flash it with.

**Comment by c0mplex1** [comment]
The download link is there on the page...
e.g.
2. Download latest deCONZ firmware. Check http://deconz.dresden-elektronik.de/deconz-firmware for the latest release and replace the filename accordingly.

**Reply by Vandborg** [reply]
oh, so I don't have to flash it with at custom firmware? I thought I had to install a custom firmware for zigbee2mqtt to work. :)

**Reply by roumano** [reply]
no, it's working with zigbee2mqtt without any modification of specific firmware

**Reply by LaHoshi** [reply]
@roumano & @Vandborg which firmware version do you have in use?

i have tried `0x266b0700` and the new `0x266f0700`, but without success.

**Comment by LaHoshi** [comment]
i have a general question regardless of the firmware

did you get the conbee II stick to work? 
I've been trying for a few days and can't get any further

**Reply by dylan09** [reply]
Yes,
works perfect on an Raspberrry PI 4, 8 GByte. In my opinion the Conbee II is faster than the zzh! and the Z-Bee sticks with 2652 chip. My Aquara motion sensors respond very fast with the Conbee stick. Lights are swichted on instantly when motion is detected.
With the other sticks me and my wife feel there is a short lag (a few 100 msec).

Use a USB cable to connect the stick if there is a USB3 port nearby. As recommended  by the manufacturer.

**Reply by roumano** [reply]
> i have a general question regardless of the firmware
> 
> did you get the conbee II stick to work?
> I've been trying for a few days and can't get any further

All version of firmware tested (even the default one) (tested 4 version) are working out of the box so your problem is not here...

**Comment by roumano** [comment]
Hi @LaHoshi  yes the conbee II is working fine since few months

**Reply by LaHoshi** [reply]
Thank you @roumano !!!
now I have to do a little more research and long nights with playing trial and error...

The "Zigbee Home Automation" and "deCONZ" integration i get up und runinng, but not zigbee2mqtt.
i use a Home Assistant VM inside of Proxmox.

**Reply by RMCob** [reply]
Can't have deCONZ and zigbee2mqtt running at the same time. Have to kill deCONZ before starting zigbee2mqtt

**Comment by c0mplex1** [comment]
I experience the exact opposite as @dylan09 with a ConBee II stick.
I'm now using the recommended Slaesh's CC2652RB stick.
This one works much better and more reliable for me.
I'm glad I switched sticks.

**Comment by dylan09** [comment]
Did someone know where I could find release notes for ConBee II firmware?
In the official firmware repository only the images are available for download.

Currently I am on firmware 0x26580700 from May 2020. But didn't want to update until I know, what has changed.

**Comment by roumano** [comment]
@dylan09 , changelog have been added recently : https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Firmware-Changelog

**Reply by dylan09** [reply]
Thanks.

**Comment by civarson** [comment]
Wondering if anyone can help me. I recently migrated my Home Assistant setup from a VM on Ubuntu to a VM in Unraid. I also moved my Conbee II stick over and assured USB passthrough was enabled. For the life of me I can't get z2m in HA to start with the stick - I get the unable to start zigbee herdsman error. Installing deconz and seeing if it detects the stick actually works fine. But I really want to continue using z2m and not migrate back to deconz.
